furya/chain.json

```diff
@@ -28,20 +28,20 @@
     ]
   },
   "codebase": {
-    "git_repo": "https://github.com/furysport/furya-chain",
-    "recommended_version": "v1.0.0",
+    "git_repo": "https://github.com/furysport/furya",
+    "recommended_version": "v3.1.0",
     "compatible_versions": [
-      "v1.0.0"
+      "v3.1.0"
     ],
     "genesis": {
       "genesis_url": "https://github.com/furysport/furya-chain/blob/main/network/mainnet/genesis.json"
     },
     "versions": [
       {
-        "name": "v1.0.0",
-        "recommended_version": "v1.0.0",
+        "name": "v3.1.0",
```

Comment:
Instead of replacing v1 with v3 inside the `versions` array, please add v3 as a new object below v1. This allow us to build up a version history.